src/seat/tablet.rs

```diff
@@ -0,0 +1,90 @@
+use std::mem;
+use std::sync::Mutex;
+
+use wayland_client::{
+    Connection,
+    Dispatch,
+    QueueHandle,
+};
+use wayland_protocols::wp::tablet::zv2::client::zwp_tablet_v2::{self, ZwpTabletV2};
+
+pub trait Handler: Sized {
+    /// This is fired at the time of the `zwp_tablet_v2.done` event,
+    /// and collects any preceding `name`, `id` and `path` events into an [`Info`].
+    fn info(
+        &mut self,
+        conn: &Connection,
+        qh: &QueueHandle<Self>,
+        tablet: &ZwpTabletV2,
+        info: Info,
+    );
+
+    /// Sent when the tablet has been removed from the system.
+    /// When a tablet is removed, some tools may be removed.
+    ///
+    /// This method is responsible for running `tablet.destroy()`.  ← TODO: true or not?
```

Comment:
It would be a good time to destroy the object, but I suppose it isn't strictly responsible for it.